firefox-webext/background.js

```diff
@@ -5,6 +5,26 @@
 var prefix = 'https://webcompat.com/?open=1&url=';
 var screenshotData = '';
 
+chrome.contextMenus.create({
+  id: "webcompat",
+  title: "Report Website compatibility issue",
+  contexts: ["all"]
+});
+
+chrome.contextMenus.onClicked.addListener(function (tab) {
```

Comment:
Yeah, like you mentioned in your comment we can make this more [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).